dRAGon/search.py

```diff
@@ -0,0 +1,68 @@
+import whoosh
+import os.path
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from whoosh.index import create_in
+from whoosh.fields import * # TODO: Get rid of this for the love of god and all attendant bodhisattvas
+from whoosh.qparser import QueryParser
+from whoosh.index import open_dir
+
+
+## Define search method
+
+def search_rulebook(query=""):
+    ix = open_dir("indexdir")
+    with ix.searcher() as searcher:
+        query = QueryParser("content", ix.schema).parse(query)
+        results = searcher.search(query)
+        return results[0]
+
+
+
+
+## Initialize chunkers
+
+text_splitter = RecursiveCharacterTextSplitter(
```

Comment:
Is this the kind of thing that we want to have the ability to parameterize when run, or should the parameters be hard coded?

Reply:
I think for now we want to have this hard coded--though config for setting the text splitter to certain settings would make sense, hm.